woocommerce/woocommerce-android
# Design feedback order creation

### Why
This PR addresses the feedback received from design related to the order creation screen
More context here: pe5pgL-42P-p2#comment-3267

### Description
![image](https://github.com/woocommerce/woocommerce-android/assets/18119390/01a013bc-9776-4b9e-98b0-0416b180b5b9)

Order creation screen: the screen on the left of the image

From the design review:

> #### On the order details screen
> 1. Wrap all the bundle product cards in one – so, reduce the gaps between them to zero, and only have rounded corners on the first and last.
> 2. Keep a solid, continuous bottom border between the bundle product and its children.
> 3. Use indented (left and right) bottom borders between all the children products (unless the child is in expanded state).
> 4. Use a standard 56×56pt thumbnail for the bundle product, and indented 40×40pt thumbnails for the children.
> 5. For the children, reduce the product name font size to Subheadline (same as the text below it), a change the chevron colour to grey, and if not using individual prices, set the $0.00 price to grey.
> 6. For the children in expanded states, don’t show the “Remove product from order” button.
Identify the parent product as a Bundle (same as in the product selector).

### Testing instructions


1. Open the orders screen
2. Tap the create order (+)
3. Select a bundle product
4. Configure the product and add it to the order
5. Check that the bundles (the product and the children appears on a single card)
6. Check that the bundle and the children can be expanded
7. Check that when the children product price is $0.00, the price color is gray (when the product is collapsed) 
8. Check that children product expanded card doesn't show any 


### Images/gif

https://github.com/woocommerce/woocommerce-android/assets/18119390/17788a6e-6647-401d-9e94-e1664dea0c81
